server/test/debian_10_event_log.yml

```diff
@@ -0,0 +1,164 @@
+// Fetched via `tpm2_eventlog debian_10_binary_bios_measurements`
```

Comment:
Let's just remove this for now, as we do not use it in our tests.

Reply:
Done.